cmd/fsi.go

```diff
@@ -20,8 +20,8 @@ func NewFSICommand(f Factory, ioStreams ioes.IOStreams) *cobra.Command {
 	}
 
 	link := &cobra.Command{
-		Use:   "link",
-		Short: "link a .qri-ref",
+		Use:   "link DATASET PATH",
+		Short: "link a dataset to a directory on disk",
 		Example: `link a dataset to the current working directory:
   $ qri fsi link peername/dataset .`,
 		Args: cobra.MinimumNArgs(2),
```

Comment:
In practice, it doesn’t seem like providing multiple refs makes sense here, so maybe this should be `ExactArgs(2)`. (Also, maybe it should set the `allowed` arg to `1` instead of `-1` when it calls `GetCurrentRefSelect()`?) Wasn’t sure, though, so didn’t change it.